include/psp2kern/kernel/ssmgr.h

```diff
@@ -42,6 +42,19 @@ int ksceSblAimgrGetConsoleId(SceConsoleId *cid);
 int ksceSblAimgrGetOpenPsId(SceOpenPsId *open_psid);
 int ksceSblAimgrGetPscode(ScePsCode *pscode);
 
+int ksceSblDmac5AesCbcDec(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+int ksceSblDmac5AesCbcEnc(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+int ksceSblSsMgrAesCtrDecrypt(const void *src, void *dst, int size, const void *key, int key_size, void *iv, int mask_enable);
+
+#define ksceSblDmac5AesCtrDec ksceSblSsMgrAesCtrDecrypt
```

Comment:
why did you define it?

Reply:
It's registered in db.yml with ksceSblSsMgrAesCtrDecrypt, but obviously it's ksceSblDmac5AesCtrDec.

Reply:
hm... okay. first time i thought it's good name than before. but when we try to change it, we have to change every `ksceSblSsMgr*Decrypt/Encrypt`  functions in db.

Reply:
yes, true.
maybe i'll do some function check.